code/src/components/LoadingPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const LoadingPage = () => {
+    return ( 
+        <div className='loading-container'>
+            <div className='loading'></div>
+        </div>
+    )
+}

```

Comment:
My internet is too fast for me to see your loading page 😢 I was very curious about what it looks like.